pkg/kubecost/allocation.go

```diff
@@ -1879,17 +1843,16 @@ func (as *AllocationSet) Map() map[string]*Allocation {
 		return map[string]*Allocation{}
 	}
 
-	return as.Clone().allocations
+	return as.Clone().Allocations
```

Comment:
I'm not so sure this function is necessary. It's also really confusing and expensive.

Reply:
Let's nuke it. It shouldn't be necessary, now that the field is public.